sql/core/src/test/resources/sql-tests/results/view-schema-binding.sql.out

```diff
@@ -0,0 +1,3081 @@
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+SELECT * FROM v
+-- !query schema
+struct<c1:int,c2:int>
+-- !query output
+
+
+
+-- !query
+DESCRIBE EXTENDED v
+-- !query schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query output
+c1                  	int                 	                    
+c2                  	int                 	                    
+                    	                    	                    
+# Detailed Table Information	                    	                    
+Catalog             	spark_catalog       	                    
+Database            	default             	                    
+Table               	v                   	                    
+Created Time [not included in comparison]
+Last Access [not included in comparison]
+Created By [not included in comparison]
+Type                	VIEW                	                    
+View Text           	SELECT * FROM t     	                    
+View Original Text  	SELECT * FROM t     	                    
+View Schema Mode    	BINDING             	                    
+View Catalog and Namespace	spark_catalog.default	                    
+View Query Output Columns	[c1, c2]
+
+
+-- !query
+DROP TABLE t
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CREATE TABLE t(c1 INT) USING PARQUET
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+SELECT * FROM v
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+{
+  "errorClass" : "INCOMPATIBLE_VIEW_SCHEMA_CHANGE",
+  "sqlState" : "51024",
+  "messageParameters" : {
+    "actualCols" : "[]",
+    "colName" : "c2",
+    "expectedNum" : "1",
+    "suggestion" : "CREATE OR REPLACE VIEW spark_catalog.default.v AS SELECT * FROM t",
+    "viewName" : "`spark_catalog`.`default`.`v`"
+  }
+}
+
+
+-- !query
+DESCRIBE EXTENDED v
+-- !query schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query output
+c1                  	int                 	                    
+c2                  	int                 	                    
+                    	                    	                    
+# Detailed Table Information	                    	                    
+Catalog             	spark_catalog       	                    
+Database            	default             	                    
+Table               	v                   	                    
+Created Time [not included in comparison]
+Last Access [not included in comparison]
+Created By [not included in comparison]
+Type                	VIEW                	                    
+View Text           	SELECT * FROM t     	                    
+View Original Text  	SELECT * FROM t     	                    
+View Schema Mode    	BINDING             	                    
+View Catalog and Namespace	spark_catalog.default	                    
+View Query Output Columns	[c1, c2]
+
+
+-- !query
+SET spark.sql.viewSchemaBindingMode
```

Comment:
why do we check the config value here? We already checked the default value at the beginning.

Reply:
Will rework the QA.